File: app/src/main/java/com/khmelenko/lab/travisclient/network/response/Request.java
package com.khmelenko.lab.travisclient.network.response;

import com.google.gson.annotations.SerializedName;

/**
 * Dao for Request response
 *
 * @author Dmytro Khmelenko
 */
public final class Request {

    @SerializedName("request")
    private RequestData mRequestData;

    @SerializedName("commit")
    private Commit mCommit;

    public RequestData getRequestData() {
        return mRequestData;
    }

    public void setRequestData(RequestData request) {
        mRequestData = request;
    }

    public Commit getCommit() {
        return mCommit;
    }

    public void setCommit(Commit commit) {
        mCommit = commit;
    }
}

File: app-v3/src/main/java/com/khmelenko/lab/varis/network/response/Request.kt
package com.khmelenko.lab.varis.network.response

import com.google.gson.annotations.SerializedName

/**
 * Dao for Request response
 *
 * @author Dmytro Khmelenko
 */
data class Request (

    @SerializedName("request")
    val requestData: RequestData? = null,

    @SerializedName("commit")
    val commit: Commit? = null
)
